packages/admin/admin/src/FinalForm.tsx

```diff
@@ -235,6 +235,7 @@ export function FinalForm<FormValues = AnyObject>(props: IProps<FormValues>) {
                     }
 
                     onAfterSubmit(values, form);
+                    editDialogFormApi?.onAfterSave?.();
```

Comment:
I'm not sure if this should be called here, or in the default `onAfterSubmit` handler. The onAfterSubmit handler may be overridden, which would result in the `onAfterSave` callback not being called (only when submitting by hitting enter, click on the save button probably still works). What do you think?

Reply:
good here I'd say